src/template.c

```diff
@@ -47,20 +47,21 @@ int tmpl_parse(struct template *vars, char *dst, size_t dst_len, const char *src
 			continue;
 		}
 
-		/* we know it's a '$'. But we are interest in the next char */
```

Comment:
Let's fix typos in a separate commit.

Reply:
Maybe, but I would prefer to fix little typos in comments as I spot them, otherwise I am sure to forget them later.